dashboard/db/migrate/20210103234601_convert_teacher_feedback_to_utf8mb4.rb

```diff
@@ -0,0 +1,10 @@
+class ConvertTeacherFeedbackToUtf8mb4 < ActiveRecord::Migration[5.1]
+  def up
+    execute 'alter table teacher_feedbacks convert to charset utf8mb4 collate utf8mb4_unicode_ci'
+    change_column :teacher_feedbacks, :comment, :text, limit: 65535
```

Comment:
Does this statement change the charset/collation of the column?  If not, do we need to add a statement to do that?